### DISCUSSION TITLE
[Idea] Explicitly display current system user for file/folder permission errors

### DISCUSSION THREAD

**Discussion by mehov** [PrestaShop/PrestaShop#33262] [Ideas]
### The problem

Installing Prestashop. Getting permissions error:

> Unable to write to file /var/www/prestashop/index.php.temp

It would be a little more informative if it gave me the current system user right away.

### The solution

Seems like a small change: in https://github.com/PrestaShop/PrestaShop/blob/0644ffe53481fa2b97ddc2113848147076beddb2/tools/build/Library/InstallUnpacker/index_template.php#L168, change the line 168 to:

```
                    'message' => get_current_user(). ' is unable to write to file ' . getcwd() . '/index.php.temp',
```

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
Hi @mehov 

Maybe you'd like to create a Pull Request to improve this behavior?
